tools/frb_internal/lib/src/makefile_dart/release.dart

```diff
@@ -90,7 +90,7 @@ void _updateVersionInText() {
 
   for (final package in ['flutter_rust_bridge', 'flutter_rust_bridge_macros']) {
     simpleReplaceFile(
-      '${exec.pwd}frb_codegen/assets/integration_template/REPLACE_ME_RUST_CRATE_DIR/Cargo.lock.template',
+      '${exec.pwd}frb_codegen/assets/integration_template/shared/REPLACE_ME_RUST_CRATE_DIR/Cargo.lock',
```

Comment:
nit: do not use Cargo.lock, but use Cargo.lock.template. IIRC if we use cargo.lock, when publishing rust package, cargo will be confused and *remove* the files from our template, then we do not have this file!

Reply:
> I'm curious as to why publishing a rust crate would remove a Cargo.lock file in a different part of the file hierarchy that is not referenced from the root Cargo.toml.

I don't know, maybe cargo publish or crates.io has some logic?

Reply:
But anyway, let's keep it the old way (Cargo.lock.template). At the beginning I used Cargo.lock and later it has the trouble I mentioned thus I changed to .template ...

Reply:
Sure. I reverted the change. But going off my last concern, maybe I'm missing some logic, but won't the file get copied over as `Cargo.lock.template` when creating a project and since it ends with `.template` Cargo will ignore it and create a new lock file?